packages/bento-design-system/src/Table/cells.tsx

```diff
@@ -62,7 +78,9 @@ export function TextWithIconCell({
     text: LocalizedString;
     tooltipContent?: Children;
   }
->) {
+> & {
+  options: Partial<Pick<BodyProps, "size" | "weight" | "color">>;
+}) {
   const icon_ = icon && icon({ size: 12 });
```

Comment:
doesn't it make sense to configure also the icon size?

Reply:
yes it does, pushed a change